tutorials/README.md

```diff
@@ -9,11 +9,9 @@
 | [`ner_drugs`](ner_drugs) | Detecting drug names in online comments (Named Entity Recognition) |
 | [`ner_fashion_brands`](ner_fashion_brands) | Detecting fashion brands in online comments (Named Entity Recognition) |
 | [`ner_food_ingredients`](ner_food_ingredients) | Analyzing how mentions of ingredients change over time (Named Entity Recognition) |
-| [`ner_multiple_trials`](ner_multiple_trials) | Training a named-entity recognition (NER) with multiple trials |
 | [`ner_pytorch_medical`](ner_pytorch_medical) | Detecting entities in Medical Records with PyTorch |
 | [`ner_tweets`](ner_tweets) | Detecting people entities in tweets (Named Entity Recognition) |
-| [`parser_low_resource`](parser_low_resource) | Training a POS tagger and dependency parser for a low-resource language |
 | [`rel_component`](rel_component) | Example project of creating a novel nlp component to do relation extraction from scratch. |
-| [`spancat_food_ingredients`](spancat_food_ingredients) | Span Categorization in Prodigy |
 | [`textcat_docs_issues`](textcat_docs_issues) | Predicting whether a GitHub issue is about documentation (Text Classification) |
-| [`textcat_goemotions`](textcat_goemotions) | Categorization of emotions in Reddit posts (Text Classification) |
\ No newline at end of file
+| [`textcat_goemotions`](textcat_goemotions) | Categorization of emotions in Reddit posts (Text Classification) |
+| [`spancat_food_ingredients`](spancat_food_ingredients) | Span Categorization in [Prodigy](https://prodi.gy/) |
```

Comment:
This needs to be updated again, double-check all the category READMEs again?

Reply:
Ran `update_category_docs()`. Updated in https://github.com/explosion/projects/pull/121/commits/8e9fae061b5e3800f9bc430135b87784bba0b4ab.